altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
Great, that would be really helpful. It should be pretty simple for the
other renderers like matplotlib to render compositions - would be nice to
use that to test out the composition APIs.

On Thu, Oct 15, 2015 at 12:34 PM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> In altair/api.py
> <https://github.com/ellisonbg/altair/pull/68#discussion_r42147740>:
>
> > @@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
> >              self.shape._infer_type(self.parent.data)
> >
> >
> > +class Filter():
>
> Right after we release 0.8, we will start drafting syntax for composition
> (especially layer) so we can have a standard even before we get to
> implement them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ellisonbg/altair/pull/68/files#r42147740>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com